queries.sql

```diff
@@ -0,0 +1,9 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE (neutered = true) AND (escape_attempts < 3);
+SELECT date_of_birth FROM animals WHERE (name = 'Agumon') OR (name = 'Pikachu');
+SELECT name , escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] Also as required in the requirements, please consider 
- finding all animals that are neutered.